newm-server-portal/src/main/kotlin/io/projectnewm/server/portal/model/Song.kt

```diff
@@ -0,0 +1,19 @@
+package io.projectnewm.server.portal.model
+
+import kotlinx.datetime.LocalDate
+import kotlinx.datetime.serializers.LocalDateIso8601Serializer
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class Song(
+    @SerialName("id") val id: Int,
+    @SerialName("name") val name: String,
+    @SerialName("description") val description: String,
+    @SerialName("genre") val genre: String,
```

Comment:
Should this be an array? I could see a song having multiple genres

Reply:
> Should this be an array? I could see a song having multiple genres

Agreed

Reply:
Changed to array